apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -4937,7 +4937,7 @@ force_progress_triggers_slash(Cfg) ->
     wait_for_signed_transaction_in_block(dev1, SignedFP, Debug),
 
     _Tx = await_on_chain_report(I4, #{info => can_slash}, ?TIMEOUT),
-    _Tx = await_on_chain_report(R4, #{info => can_slash}, ?TIMEOUT),
+    _Tx1 = await_on_chain_report(R4, #{info => can_slash}, ?TIMEOUT),
```

Comment:
Hmm, these _should_ actually be the same tx, so removing the '_' is perhaps more appropriate.

Reply:
Nicely spotted. Will change